content/en/docs/v3.5/tutorials/how-to-setup-cluster.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How to setup an etcd cluster
```

Comment:
I'd like to retitle this something that indicates that this is a demo cluster.  Probably as simple as "How to Set Up a Demo Etcd Cluster".

Reply:
... because this is not a production setup.